### PR TITLE
inotify-tools 3.22.1.0 (new formula)

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
The `gitwatch` formula uses `fswatch` on macOS and `inotify` on Linux. This provides the missing formula.

I was not able to get the test to run, it should display:
```
Establishing watches...
Finished establishing watches, now collecting statistics.
```

But as the program never returns the popen step gets stuck. How do we handle this in ruby?

File: Formula/inotify-tools.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  sha256 "da81010756866966e6dfb1521c2be2f0946e7626fa29122e1672dc654fc89ff3"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on :linux
+
+  def install
+    inreplace "libinotifytools/src/inotifytools.c", "#ifdef __FreeBSD__", "#ifdef __APPLE__"
```
Comment (carlocab) [inline Formula/inotify-tools.rb:14]: This seems like a strange `inreplace` for a Linux-only formula

Comment (iMichka) [reply Formula/inotify-tools.rb:14]: Ha ha yes let me remove it, I was trying to fix it on macOS.
That inreplace helped, I was able to compile it, but it was not usable on macOS.
I did not have time to investigate more, so I made the formula linux only.

Looks like upstream is active so they might fix it on macOS one day.

File: Formula/inotify-tools.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}",
+                          "--mandir=#{man}",
+                          "--disable-dependency-tracking"
+    system "make", "install"
+  end
+
+  test do
+    touch "test.txt"
+    stdin, stdout, stderr, _ = Open3.popen3("#{bin}/inotifywatch test.txt --timeout 2")
```
Comment (carlocab) [inline Formula/inotify-tools.rb:23]:
```suggestion
    stdin, stdout, stderr, = Open3.popen3("#{bin}/inotifywatch test.txt --timeout 2")
```

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**